src/map/npc.cpp

```diff
@@ -5294,20 +5294,29 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		}
 		mob.xs = 0;
 	}
```

Comment:
Further below there is a case where x and y are 0 it would just put xs and ys to -1 (map-wide).

Reply:
Hmm, the logic of this function is that of x and y is 0 then the coordinates are map-wide xs and ys don't matter in this case. There actually isn't a way to set xs and ys so that the coordinates are random (the default value and any given values will overwritten with 0), it just depends on x and y. 

Though I'm considering to changing the min cap to -1 instead based on your other comment. Unfortunately you cannot intentionally set it to -1 as there will be a warning issued then.

I guess we could issue a warning if the center is (0,0) and xs or ys are above 0 that the spawn will be random instead?